<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>763</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="transliterateImageButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>130</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Transliterate Image</string>
   </property>
  </widget>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>0</y>
     <width>351</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Baybayin Transliteration Tool</string>
   </property>
  </widget>
  <widget class="QPushButton" name="captureImageButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>30</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Capture and Save Image</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startStopCameraButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>410</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Close Application</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>431</width>
     <height>331</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="cameraLayout">
    <item>
     <widget class="QLabel" name="cameraLabel">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>640</width>
        <height>480</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>260</y>
     <width>241</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QTextEdit" name="outputText"/>
    </item>
   </layout>
  </widget>
  <widget class="QTextEdit" name="ocrOutputBox">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>180</y>
     <width>239</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="binarizeImageButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>80</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Binarize Image</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
